### DISCUSSION TITLE
Does BoxStarter has Hotfixes? (Bugs were fixed > 1.5 years ago, but are still not in the release 😞)

### DISCUSSION THREAD

**Discussion by schittli** [mwrock/boxstarter#559] [Q&A]
Good morning

I have a small question:

1. BoxStarter has the extremely annoying issue for more than a year an endless number of verbose messages are displayed
2. This Bug was fixed > 1.5 years ago:
    #512
3. Because the latest BoxStarter release still has this issue, I assume that the bug fix has still not been released 😞 

Therefore my question:
Does BoxStarter has hotfixes, which can easily be installed?

Thanks a lot, kind regards,
Thomas

**Comment by pauby** [comment]
Just to be clear, Boxstarter is a community maintained open-source project. All of your expectations should be guided by that.

In addition to that, if you have issues, and you can't wait for them to be resolved, you're expected to get your hands dirty and help out.  

I mention the above as your message comes across frustrated, angry and annoyed _to me_.

> This Bug was fixed > 1.5 years ago: https://github.com/chocolatey/boxstarter/pull/512

According to what I can see, this was released in 3.0.1. @gep13 and / or @mwallner can confirm or deny that.

> Because the latest BoxStarter release still has this issue, I assume that the bug fix has still not been released 😞

Perhaps you are seeing another issue. @mwallner did say in [this comment](https://github.com/chocolatey/boxstarter/pull/512#issuecomment-1291083531) that he is working on other issues, but it's not clear if these are the same issues as you're seeing or others ('defect' was mentioned which makes me think it's not this type of issue).

**Comment by mwallner** [comment]
Good evening :wink:

first, 100% what pauby said.
second, you did the right thing reaching out - although the [Discord Server](https://ch0.co/community) might have been a better choice in this particular case.

The issue you're referring to has indeed been fixed a while back and afaik there has not been a regression. I'd be happy to look into it, please provide a sample of what you're seeing (link to a gist?) - if there's been a regression I'd be more than happy to open an issue and have it fixed in the upcoming [3.1.0 release](https://github.com/chocolatey/boxstarter/milestone/8).


Just to make sure we're on the same page: that's about the amount of output the currently latest released version of Boxstarter (3.0.3) produces:

```
PS C:\Users\flynn> BoxstarterShell.ps1
Welcome to the Boxstarter shell!
The Boxstarter commands have been imported from C:\ProgramData\Boxstarter and are available for you to run in this shell.
You may also import them into the shell of your choice.

Here are some commands to get you started:
Install a Package:   Install-BoxstarterPackage
Create a Package:    New-BoxstarterPackage
Build a Package:     Invoke-BoxstarterBuild
Enable a VM:         Enable-BoxstarterVM
For Command help:    Get-Help <Command Name> -Full

For Boxstarter documentation, source code, to report bugs or participate in discussions, please visit https://boxstarter.org
PS C:\> Install-BoxstarterPackage wiztree -DisableReboots
Boxstarter: Installing package wiztree
Boxstarter Version 3.0.3.0
(c) 2018 Chocolatey Software, Inc, 2012 - 2018 Matt Wrock. https://boxstarter.org

Boxstarter: Disabling Automatic Updates from Windows Update
+ Boxstarter starting Calling Chocolatey to install wiztree. This may take several minutes to complete...
Chocolatey v2.2.2
Installing the following packages:
wiztree
By installing, you accept licenses for the packages.
Progress: Downloading wiztree 4.18.0... 100%

wiztree v4.18.0 (forced) [Approved]
wiztree package files install completed. Performing other installation steps.
 *** LOADING BOXSTARTER ***
Boxstarter: *** HELLO WORLD ***
Installing wiztree...
wiztree has been installed.
  wiztree can be automatically uninstalled.
 The install of wiztree was successful.
  Software installed to 'C:\Program Files\WizTree\'

Chocolatey installed 1/1 packages.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Did you know the proceeds of Pro (and some proceeds from other
 licensed editions) go into bettering the community infrastructure?
 Your support ensures an active community, keeps Chocolatey tip-top,
 plus it nets you some awesome features!
 https://chocolatey.org/compare
+ Boxstarter finished Calling Chocolatey to install wiztree. This may take several minutes to complete... 00:00:12.5362781
True
Boxstarter: Restore Automatic Updates from Windows Update


Errors       : {}
ComputerName : localhost
Completed    : True
FinishTime   : 09.02.2024 20:54:05
StartTime    : 09.02.2024 20:53:51
```
(Note that I've got Chocolatey 2.2.2 installed on this particular box, the output of a 'Boxstarter-only' setup with the bundled version of choco should be pretty much the same - just featuring choco 1.1, the already mentioned upcoming release of Boxstarter will include 2.2.2 anyway :wink: )
